.lintstagedrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "**/*.{js,jsx,ts,tsx}": [
+    "eslint --max-warnings=0",
+    "prettier --write",
+    "vitest related --run"
```

Comment:
The commands in package.json use cross-env + TZ set to Helsinki, maybe here too so that they'd be as similar as possible? I'm not sure whether the TZ setting is needed in either place, but it might be to make the running of tests more consistent/predictable.